Update cip-0000.md (#92)
global-synchronizer-foundation/cips: cip-0000/cip-0000.md

- Send the text of the CIP to the CIP Discuss mailing list
  - Work with a CIP editor for refinement and finalize as a Pull Request
  - Iterate until the CIP has a sponsor and endorser on [cip-discuss](https://lists.sync.global/g/cip-discuss/topics)

> - Send your contact info to operations@sync.global, including at least email and a link to the CIP pull request.

Author: ```suggestion
- Send contact info for all parties involved in the CIP to operations@sync.global, including at least email addresses and a link to the CIP.
```
